### ISSUE TITLE
License

### ISSUE THREAD

**Issue by contrapunctus-1** [vseloved/wiki-lang-detect#5]
Currently, this project is listed on https://github.com/CodyReichert/awesome-cl, but with "no license specified".

Please consider adding a file called LICENSE or COPYING containing the license text. I'd also suggest mentioning the license in the README, and maybe linking to the license file there.

**Comment by rbugajewski** [comment]
Indeed, it would be great to know if all work in this repository is under copyright and for educational purposes only, or if no license specified means this is in the public domain.

A clarification would be very kind.
